src/components/MAPI/workernodes/WorkerNodesAzureMachinePoolSpotInstances.tsx

```diff
@@ -44,10 +44,10 @@ const WorkerNodesAzureMachinePoolSpotInstances: React.FC<IWorkerNodesAzureMachin
               }-spot-instances-tooltip`}
             >
               <Box width='small'>
-                <Text size='small'>{headline}</Text>
-                {maxPriceRow && (
-                  <Text size='small' margin={{ top: 'xsmall' }}>
-                    {maxPriceRow}
+                <Text size='xsmall'>{headline}</Text>
+                {maxPriceText && (
+                  <Text size='xsmall' margin={{ top: 'xsmall' }}>
+                    {maxPriceText}
```

Comment:
Looks like something we shouldn't have to adjust each time. I'd expect all tooltips to have their text in the same size.